js/config.js

```diff
@@ -69,13 +89,13 @@ var respecConfig = {
     github: "https://github.com/Logius-standaarden/OAuth-NL-profiel/",
     issueBase: "https://github.com/Logius-standaarden/OAuth-NL-profiel/issues",
     licence: "cc-by-nd",
-    previousPublishDate: "2020-07-09",
-    previousPublishVersion: "v1.0",
+    previousPublishDate: "2024-12-03",
+    previousPublishVersion: "1.1.0",
```

Comment:
```suggestion
    previousPublishVersion: "v1.1.0",
```
Om naar de bestaande permalink te verwijzen. Komende releases lekker zonder `v`